### PR TITLE
Agency seal and list item c d

### PR DESCRIPTION
**Pull request by hferraio**
addresses 2 issues.

### CODE REVIEW

File: _FIPS201/frontend.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ Zone 9F: Header
 {:latex-literal="true"}
 
 Zone 11F: Agency Seal
-: If used, the seal selected by the issuing department, agency, or organization
+: The Agency seal in 11F **MAY** become a mandatory field in the next revision of the Standard. If used, the seal selected by the issuing department, agency, or organization
```
Comment (jimfenton) [inline _FIPS201/frontend.md:298]:
```suggestion
: The Agency seal in 11F may become a mandatory field in the next revision of the Standard. If used, the seal selected by the issuing department, agency, or organization
```

**Review by jimfenton — CHANGES_REQUESTED** [review]
Non-normative "may"

**Review by jimfenton — APPROVED** [review]
looks good now

File: _FIPS201/requirements.md [hunk 2]
```diff
@@ -25,8 +25,7 @@ electronically; and (d) is issued only by providers whose reliability has been e
 accreditation process.
 
 
-Each agency's PIV implementation **SHALL** meet the four control objectives (a) through (d) listed above
-such that
+Each agency's PIV implementation **SHALL** meet the control objectives listed above to include, but not limited to processes that ensure that
```
Comment (jricher) [inline _FIPS201/requirements.md:28]:
```suggestion
Each agency's PIV implementation **SHALL** meet the control objectives listed above to include, but to not be limited to, processes that ensure that
```

File: _FIPS201/frontend.md [hunk 3]
```diff
@@ -295,7 +295,7 @@ Zone 9F: Header
 {:latex-literal="true"}
 
 Zone 11F: Agency Seal
-: If used, the seal selected by the issuing department, agency, or organization
+: The Agency seal in 11F may become a mandatory field in the next revision of the Standard. If used, the seal selected by the issuing department, agency, or organization
```
Comment (jricher) [inline _FIPS201/frontend.md:298]:
```suggestion
: The Agency seal in Zone 11F may become a mandatory field in the next revision of the Standard. If used, the seal selected by the issuing department, agency, or organization
```

**Pull request merged**